dll/win32/syssetup/lang/zh-CN.rc

```diff
@@ -30,16 +30,16 @@ BEGIN
 END
 
 IDD_PRODUCT DIALOGEX 0, 0, 317, 143
-CAPTION "ReactOS Setup"
+CAPTION "ReactOS 安装程序"
 STYLE DS_MODALFRAME | DS_SHELLFONT | WS_POPUPWINDOW | WS_CAPTION | WS_VISIBLE
-FONT 8, "MS Shell Dlg"
+FONT 9, "宋体"
 BEGIN
     ICON "", IDC_PRODUCT_ICON, 5, 5, 20, 20
-    LTEXT "Please choose a product option:", IDC_STATIC, 35, 7, 230, 12
-    LTEXT "Product &Options:", IDC_STATIC, 5, 32, 85, 10
+    LTEXT "请选择您要安装的软件产品:", IDC_STATIC, 35, 7, 230, 12
```

Comment:
请选择产品类型:

Reply:
这个翻译是参照繁体中文得来的，私以为按照意译翻译成“请选择您要安装的软件产品”更人性化一点。

Reply:
This translation result refers to the corresponding translation of the traditional Chinese version, which I think is more humane.

Reply:
This translation result refers to the corresponding translation of the traditional Chinese version, which I think is more humane.

Reply:
```suggestion
    LTEXT "请选择您要安装的软件版本：", IDC_STATIC, 35, 7, 230, 12
```

* As the translator of Traditional Chinese, I have paraphrased "product" to "version" (today) because "產品" is quite weird in the context of sentence for Chinese.
  * 我覺得這裡應該是「版本」才符合整個 string 的意境。我當時翻譯的時候沒有很好的了解句子的意境。
* There is another translator who is translating these lines (PR #3933). A communication with him is recommended as strings of PR which merged later will replace the one which merge earlier.

Reply:
```suggestion
    LTEXT "请选择您要安装的软件版本：", IDC_STATIC, 35, 7, 230, 12
```

* As the translator of Traditional Chinese, I have paraphrased "product" to "version" (today) because "產品" is quite weird in the context of sentence for Chinese.
  * 我覺得這裡應該是「版本」才符合整個 string 的意境。我當時翻譯的時候沒有很好的了解句子的意境。
* There is another translator who is translating these lines (PR #3933). A communication with him is recommended as strings of PR which merged later will replace the one which merge earlier.
> Yes I repost this comment (if you saw it already) because the comment is duplicated.

Reply:
> 请选择产品类型: or 请选择产品选项:

我再完善下吧。